frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -10,17 +10,6 @@ const cephStorageLabel = 'cluster.ocs.openshift.io/openshift-storage';
 export const filterCephAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'annotations.storage_type') === 'ceph');
 
-export const getCephPVCs = (
-  cephSCNames: string[] = [],
-  pvcsData: K8sResourceKind[] = [],
-): K8sResourceKind[] =>
-  pvcsData.filter((pvc) =>
-    cephSCNames.includes(
-      _.get(pvc, 'spec.storageClassName') ||
-        _.get(pvc, ['metadata', 'annotations', 'volume.beta.kubernetes.io/storage-class']),
```

Comment:
` _.get(pvc, ['metadata', 'annotations', 'volume.beta.kubernetes.io/storage-class']),`
reason for removing this.
this was taken care of as a bug fix.

Reply:
Yes but the approach is changed now to use refLinks from PV's. Please read the bug description for why we are totally relying on it.

Reply:
This was introduced because few pvc can have storage class name in annotations.
Are we assured now such pvcs are not created anymore.
For unbound I think it should be done.

Reply:
storage classes are not reliable. What is the user deletes the storage class?

Reply:
ack using it now